src/queries/label-text.ts

```diff
@@ -35,13 +38,19 @@ function queryAllLabelsByText(
     })
 
     return matcher(textToMatch, label, text, matchNormalizer)
-  })
+  }) as HTMLElement[]
```

Comment:
This isn't inferrable?

Reply:
This is widening the type from `HTMLLabelElement[]`. Though I don't know why this is intended.

Reply:
Actually a nice constraint would be to add a queryAll function type that has `HTMLElement[]` as the return type. Then if we mess up the implementation to return something else, it would break.

Reply:
Returning an `HTMLLabelElement` if `HTMLElement` is expected is perfectly fine though since `HTMLLabelElement extends HTMLElement`. The other way around would be problematic.